.circleci/config.yml

```diff
@@ -287,8 +353,21 @@ jobs:
       - *setmetapackageversion
       - *builddebianpackage
 
+  build-nightly-dom0-rpm:
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - *installgitlfs
+      - *clonesecuredropworkstation
+      - *getrpmnightlyversion
+      - *buildrpm
+      - *sign_rpm
+      - *addsshkeys
+      - *commitworkstationrpms
+
 workflows:
-  build-debian-packages:
+  build-packages:
```

Comment:
I was thinking that this was worth skipping since the build logic itself is in securedrop-workstation, however: you're making a good point regarding not noticing breaking the CI logic, so worth running on a per-PR basis. the other consideration is that the actual cloning of the dev rpm lfs repository will take increasing amounts of time/bandwidth, but that's an issue we need to tackle regardless (re: removing old artifacts)